apps/backoffice-v2/src/pages/TransactionMonitoringAlerts/components/AlertsHeader/AlertsHeader.tsx

```diff
@@ -28,6 +36,53 @@ export const AlertsHeader: FunctionComponent<{
     },
     [mutateAssignAlerts, selected],
   );
+  const onMutateAlertsDecision: ComponentProps<typeof AlertsDecisionDropdown>['onDecisionSelect'] =
+    useCallback(
+      decision => () => {
+        const screamingSnakeDecision = toScreamingSnakeCase(decision);
+
+        mutateAlertsDecision({
+          decision: alertDecisionToState[screamingSnakeDecision],
+          alertIds: Object.keys(selected ?? {}),
+        });
+      },
+      [mutateAlertsDecision, selected],
+    );
+  const decisions = [
+    ...['Revert Decision' as const, ...AlertStates]
```

Comment:
Why is 'Revert Decision'  being manually added to the Alert States list?
Also please extract to a constant

Reply:
Its manually added since it does not exist on an alert as a state as it changes the state to new/triggered.